t/proj/db_routers.py

```diff
@@ -0,0 +1,26 @@
+class AlwaysSecondaryDbRouter:
```

Comment:
why always Secondary DB router?

Reply:
I removed this part and simplified the test